packages/commons-ui-core/src/ImageButton/ImageButton.js

```diff
@@ -38,14 +38,19 @@ const ImageRoot = styled("span", {
 }));
 
 const ImageButton = React.forwardRef(function ImageButton(props, ref) {
-  const { src } = props;
-  let image;
+  const { children, src, onClick } = props;
+  let image = children;
   if (src) {
     image = <ImageRoot src={src} />;
   }
   const ownerState = { ...props };
   return (
-    <ImageButtonRoot ownerState={ownerState} ref={ref} {...props}>
+    <ImageButtonRoot
+      ownerState={ownerState}
+      ref={ref}
+      {...props}
+      onClick={onClick}
```

Comment:
If we're not doing anything special with `onClick`, we can leave it as part of `props`.